webpack.config.js

```diff
@@ -0,0 +1,63 @@
+const { lstatSync, readdirSync } = require('fs');
+const path = require('path');
+
+// This function generates configuration for files in the
+// ./src/examples/ folder
+const generateExampleEntries = function () {
+  const src = './examples';
+
+  // Get all subdirectories in the ./src/apps,
+  // so we can just add a new folder there and
+  // have automatically the entry points updated
+
+  const getDirectories = source =>
+    readdirSync(source)
+      .map(name => path.resolve(source, name))
+      .filter(s => lstatSync(s).isDirectory());
+
+  const exampleDirs = getDirectories(src);
+
+  return exampleDirs.reduce((entry, dir) => {
+    entry['./' + path.basename(dir) + '/bundle'] = `${dir}/index`;
+
+    return entry;
+  }, {});
+};
+
+module.exports = {
+  mode: 'development',
+
+  entry: generateExampleEntries(),
+
+  devServer: {
+    contentBase: path.resolve(__dirname, 'examples'),
+    publicPath: '/',
+  },
+
+  output: {
+    filename: '[name].js',
+    path: path.resolve(__dirname, 'dist'),
```

Comment:
could you add this directory to `.gitignore`?

Reply:
`webpack-dev-server` uses memory instead of file system for serving the files. So no files or directories will be created in the disk. But I've added `dist` to `.gitignore` anyway. Pushing in the code shortly.